src/bootloader/rollbacks/rollback.h

```diff
@@ -10,6 +16,36 @@ class Rollback {
   virtual void setBootOK() {}
   virtual void updateNotify() {}
   virtual void installNotify(const Uptane::Target& target) { (void)target; }
+
+ protected:
+  std::string getVersion(const Uptane::Target& target) {
+    std::string file;
+    for (auto& p : boost::filesystem::directory_iterator("/ostree/deploy/lmp/deploy/")) {
```

Comment:
@igoropaniuk also notice that we will iterate over all directories looking for one with the sha256 (apparently they could be named sha.1, sha.2, sha.0.....so this covers all the cases)

Reply:
Can we have this path and the firmware/version.txt path provided via config? Or would that be too much? Just that hardcoding it here might end up biting us later.

Reply:
will change